src/utils/helpers.ts

```diff
@@ -38,11 +38,23 @@ export const gridFromUrl = (gridUrl: string) => {
   return gridUrl ? gridUrl.substring(15) : null
 }
 
-
-export function isDMP (work: WorkMetadata) {
+export function isDMP(work: WorkMetadata) {
   return work.types.resourceTypeGeneral === 'OutputManagementPlan'
 }
 
-export function isProject (work: WorkMetadata) {
-  return work.types.resourceType === 'Project' && (work.types.resourceTypeGeneral === 'Other' || work.types.resourceTypeGeneral === 'Text')
+export function isProject(work: WorkMetadata) {
```

Comment:
I think adding tests for these sort of helper functions is a good idea. Since these are units of reusable code and changes to them could affect multiple parts of the codebase. Having a spec fail, would alert me to fact that I'm changing something that perhaps has larger ramifications.

I have noticed that there are in fact no specs for the helpers.ts file so you could consider this comment suggestion for the future since I don't think this should stop you from proceeding with this work.

Reply:
Agreed. We should look at adding more unit-level tests for this project.